docs/rfc/active/graph_ql_frontend/README.md

```diff
@@ -0,0 +1,607 @@
+- Start Date: 12/17/2020
+- RFC PR: <TBD>
+- Implementation PR(s): <TBD>
+
+# GraphQL Frontend (Phase 1)
+
+## Summary
+
+This RFC outlines a proposal to implement a GraphQL specification in `datahub-frontend`. Ultimately, this proposal aims to model the following using GraphQL:
```

Comment:
Maybe also add a simple architecture diagram illustrating the relationships between different blocks mentioned here?

Reply:
Not completely following... Are you suggesting something like:

1. Reads --> 2. Writes --> 3. Search --> 4. Browse? 

To depict the sequencing of work suggested here?